Package.swift

```diff
@@ -28,6 +28,7 @@ let package = Package(
     .library(name: "SwiftSyntaxMacros", targets: ["SwiftSyntaxMacros"]),
     .library(name: "SwiftSyntaxMacroExpansion", targets: ["SwiftSyntaxMacroExpansion"]),
     .library(name: "SwiftSyntaxMacrosTestSupport", targets: ["SwiftSyntaxMacrosTestSupport"]),
+    .library(name: "SwiftLexicalScopes", targets: ["SwiftLexicalScopes"]),
```

Comment:
We should talk about the name here a little more. I think I suggested `SwiftLexicalScopes` initially, but given that the "scope" part is actually getting somewhat downplayed... should we call this `SwiftLexicalLookup`? All of these operations are, essentially, lexical lookups for various kinds of things---lookups for declarations, labeled statements, where a `fallthrough` ends up---and most will have "lookup" in their names.

Reply:
Yeah, I also like the idea. I changed it to `SwiftLexicalLookup`. It’s more broad and not limiting the library to just scope lookup could pave the way for including a broader range of APIs, like the `break` and `continue` lookup you’ve mentioned yesterday that is not necessarily part of `ASTScope` as far as I remember. Maybe it would be a good idea to use this library in the long run to consolidate this sort of logic in one codebase?

Reply:
Yes, I think this is the right place to consolidate all of the lexical lookup logic over time.